tekton/publish.yaml

```diff
@@ -146,7 +146,7 @@ spec:
       done
 
       # Publish images and create release.yaml
-      ko resolve --preserve-import-paths -f /workspace/go/src/github.com/tektoncd/pipeline/config/ > /workspace/output/bucket/latest/release.yaml
+      ko resolve --preserve-import-paths -t $(inputs.params.versionTag) -f /workspace/go/src/github.com/tektoncd/pipeline/config/ > /workspace/output/bucket/latest/release.yaml
```

Comment:
Thank you for this. 

Tags are already added today - see L199-212. 

The tagging is done using the gcloud client, and I would love to move away from it, to make this task more portable.  I would be happy to apply the tags using `ko`, but I don't think we can handle the regional tags with `ko`.

Reply:
To be honest, since we do not create regional release files, I doubt the regional images are ever used by anyone. I wonder if we should just push the images to gcr.io (which is hosted in the US for now, according to docs) - and we could move the bit that pushes to different regions to an optional step / task which is only executed when the target registry is gcr.io.
@bobcatfish @vdemeester thoughts?

Reply:
@afrittoli we could to the initial push with the tag to `gcr.io` using `ko` (so no gcloud), and do the replication with the `gcloud` binary separately (maybe in async pipeline/task even :wink:)

Reply:
@vdemeester that sounds good to me. @adshmh this patch as it is now pushes the tag twice to gcr.io. Could you please remove the gcloud one - leaving the regions. We can move the region bit to a separate task in a different PR.

Reply:
@afrittoli let's get this one merged, I'll do a quick follow-up :wink:

Reply:
Thank you for the reviews. I will submit another PR to remove the gcloud tagging.

Reply:
@adshmh see #1536 :sweat_smile: